tsdate/util.py

```diff
@@ -300,3 +300,43 @@ def add_sampledata_times(samples, sites_time):
     copy.sites_time[:] = sites_time
     copy.finalise()
     return copy
+
+
+def change_time_measure(time_ago, breakpoints, time_measure):
+    """
+    Rescales time given a piecewise-constant time measure. To convert from
+    generations to coalescent units, the time measure per epoch should be 2 *
+    effective population size.  To convert from coalescent units to
+    generations, the time measure should be the coalescent rate ``1/(2 * Ne)``.
+
+    :param np.ndarray time_ago: An increasing vector of time points
+    :param np.ndarray breakpoints: Start times of pieces
```

Comment:
We require the first breakpoint to be 0, right? So we should probably say this.

Reply:
(Or, as a I suggested in my review, maybe we should have the breakpoints array be of size one less than the population size array?)